build/charts/theia/values.yaml

```diff
@@ -99,3 +99,13 @@ grafana:
     - pod_to_external_dashboard.json
     - node_to_node_dashboard.json
     - networkpolicy_dashboard.json
+sparkOperator:
+  # -- Install the Spark Operator. It is required to run Network Policy Recommendation jobs.
+  enable: false
```

Comment:
We can do that. If so, do we also update our flow-visibility.yaml to include the Spark Operator? With this, the YAML file users need to apply spark-operator-crds.yaml together with the clickhouse operator before they apply flow-visibility.yaml. 

Would also like to hear from @salv-orlando about whether to include it by default.

Reply:
> If so, do we also update our flow-visibility.yaml to include the Spark Operator? With this, the YAML file users need to apply spark-operator-crds.yaml together with the clickhouse operator before they apply flow-visibility.yaml.

I prefer not to include the Spark Operator in `flow-visibility.yaml`, because we didn't include the clickhouse operator crd either.

Reply:
The Spark Operator case is a little bit different comparing the ClickHouse Operator, as besides CRDs, it contains some Deployment and Service Account.

If we do not include the Spark Operator in `flow-visibility.yaml`, there will be a gap that by default, Helm install includes the Spark Operator, but `flow-visibility.yaml` does not. If the YAML file users would like to enable the Spark Operator, they need apply spark-operator-crds.yaml and generate a `flow-visibility.yaml` with Spark Operator Deployment by running `./hack/generate_manifest.sh --spark-operator`.

I'm not sure whether this may make users confused.

Reply:
Got it, if so let's keep consistent with current steps. If we are moving forward to include spark operator in helm by default, let's include the spark operator deployments in flow-visibility.yaml and remind yaml file users apply the crd of both clickhouse and spark operator before applying flow-visibility.yaml

Reply:
my take is that `flow-visibility.yaml` should be for users that install Antrea the "hard way". These would likely more be users that try to get a good understand of how the various pieces work and integrate each other.
So I do not see as a must having the spark operator in flow-visibility.yaml. But being able to add it won't do any harm.

Every other user would like to install Antrea (and manage it after installation) in the simplest possible way, and they will likely go for helm charts.

Reply:
Got it, also according to our discussion in #36, it's ok to let users set mandatory parameters in `values.yaml` before running `helm install`. We could keep `sparkOperator enable: false` here by default.